api/server.go

```diff
@@ -106,7 +106,7 @@ func (s *Server) ListenAndServe() error {
 		logrus.Info("Shutting down...")
 		cancel()
 
-		if err := srv.Shutdown(ctx); err != nil {
+		if err := srv.Shutdown(context.Background()); err != nil {
```

Comment:
No timeout configured on a new context ?

Reply:
It would be tricky to have a timed shutdown after that all our backgrounds goroutines finished to stop.
I prefer that we don't Fatal because Shutdown didn't stop in a given time.